.cirrus.yml

```diff
@@ -85,6 +81,7 @@ test_linux_task:
 
   check_reproducible_script: |
     if [ -n "$CHECK_REPRODUCIBLE" ]; then
+      taskset 1
       make check_reproducible
```

Comment:
Do we run things after `make check_reproducible`? If so should we reset the affinity after this command?

Reply:
Reproducible is always the last one.